etc/pip-requires

```diff
@@ -16,3 +16,8 @@ paramiko==2.7.2                                   # SSH2 protocol library
 Flask==1.1.2                                      # Python web framework
 oic==1.2.1                                        # for authentication via OpenID Connect protocol
 prometheus_client==0.9.0                          # Python client for the Prometheus monitoring system
+python-magic>=0.4.15,<0.5.0                       # File type identification using libmagic
+futures>=3.2.0; python_version < '3.0'            # Clean single-source support for Python 3 and 2
```

Comment:
It doesn't make much sense to just include this for the server package, since this is already Python 3 only. Why is this not needed for clients anymore?

Reply:
As far as I could tell, they are not used in any of the client code. It's possible they are not used on the server either in which case they could be completely removed, but I did not fully check this.